packages/components/src/components/Modal/Modal.module.css

```diff
@@ -8,8 +8,10 @@
     calc(var(--dds-spacing-x1-5) - var(--dds-modal-content-padding))
     calc(var(--dds-spacing-x1-5) - var(--dds-modal-content-padding));
   display: flex;
-  flex-direction: column-reverse;
+  flex-direction: column;
   min-width: 200px;
+  max-height: calc(100vh - var(--dds-spacing-x1));
+  overflow: auto;
```

Comment:
Kan forsåvidt legge dette bak noe "scrollable" prop for å gjøre en eventuell release hakket tryggere

Reply:
Lurt, ja